plugins/org.locationtech.udig.tutorials.rcp/src/org/locationtech/udig/tutorials/rcp/MapView.java

```diff
@@ -331,4 +333,20 @@ public void setActive( ModalTool tool ){
         tool.setActive(true);
         activeTool = tool;
     }
+
+    @Override
+     public UDIGDropHandler getDropHandler() {
+        return mapviewer.getDropHandler();
+     }
+
+    // helper methods for Toolmanager
+    @Override
```

Comment:
You may use the /** comment instead of // ?

Reply:
remove comment here as well